.ci/install_rust.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Ant Financial
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+cidir=$(dirname "$0")
+rustarch=$(uname -m)
+release="nightly"
+
+if [ "${rustarch}" == "ppc64le" ]; then
+	rustarch="powerpc64le"
+fi
+
+# rust specific packages
+sudo -E apt-get update && apt-get install -y build-essential g++ \
```

Comment:
This are distro specific installs could you  move this to https://github.com/kata-containers/tests/blob/master/.ci/setup_env_ubuntu.sh, and as bonus would be nice to add to all the distros.

Reply:
cmake/musl-libc is missed/outdated in some other distro, that why ubuntu is used for rust agent.

maybe we will resolve cmake/musl-libc in other distros later, as what we do in osbuilder

Reply:
some pakages(libstdc++-8-dev) are only in bionic. It is needed to build rust agent. move this into specific distro script error out.